components/layout/flow.rs

```diff
@@ -679,6 +679,8 @@ bitflags! {
 
         /// Whether this flow contains any text and/or replaced fragments.
         const CONTAINS_TEXT_OR_REPLACED_FRAGMENTS = 0b0001_0000_0000_0000_0000_0000,
+
+        const IS_FLEX = 0b0010_0000_0000_0000_0000_0000,
     }
```

Comment:
This should be done in block flags since block is the only child type of a flex container.